### ISSUE TITLE
Allow to set the dpi of the output raster

### ISSUE THREAD

**Issue by kwinkunks** [agile-geoscience/seisplot#11]
I think we did this in `geotransect` too.

**Comment by kwinkunks** [comment]
We probably have to be a bit careful with setting output DPI in seisplot, because it uses ‘real world’ sizes for the plot — traces per inch and inches per second. I have never tested the actual output sizes of things, but I think changing the output DPI will break this — unless you compute a new internal scale to account for it.

All this might be different for PNG vs PDF (which perhaps accounts for real world plot sizes).

**Comment by EvanBianco** [comment]
Good point. I'll try this for the same 'real world' data dimensions and see if it produces the same size size plot. DPI should only be as big as it needs to be to avoid large image file sizes. Save time seems to take the most amount of time when running seisplot.
